docs/content/Coding Milestones/PoC2023q1/commands.md

```diff
@@ -540,8 +535,12 @@ Location named `demo1` with labels `foo=bar` and `the-word=the-bird`.
 This was equivalent to the following commands.
 
 ```shell
-kubectl ws root:imw-1
-kubectl create -f -<<EOF
+imw_space_config="${PWD}/temp-space-config/spaceprovider-default-imw-1"
+secret_name=$(kubectl --kubeconfig $sm_core_config --context $sm_context get space imw-1 -n spaceprovider-default -o jsonpath="{$.status.externalSecretRef.name}")
+secret_namespace=$(kubectl --kubeconfig $sm_core_config --context $sm_context get space imw-1 -n spaceprovider-default -o jsonpath="{$.status.externalSecretRef.namespace}")
+kubectl --kubeconfig $sm_core_config --context $sm_context get secret ${secret_name} -n ${secret_namespace} -o jsonpath='{$.data.kubeconfig}' | base64 -d | base64 -d > $imw_space_config
```

Comment:
These four commands are the implementation of another command that could be used instead here, right?